endpoints/openrtb2/amp_auction.go

```diff
@@ -91,18 +91,17 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	ao := analytics.AmpObject{
 		Status: http.StatusOK,
 		Errors: make([]error, 0),
+		// Prebid Server interprets request.tmax to be the maximum amount of time that a caller is willing
+		// to wait for bids. However, tmax may be defined in the Stored Request data.
+		//
+		// If so, then the trip to the backend might use a significant amount of this time.
+		// We can respect timeouts more accurately if we note the *real* start time, and use it
+		// to compute the auction timeout.
+		StartTime: time.Now(),
```

Comment:
I thought that the standalone `start` variable became redundant and just takes stack space once the same value is in the ao object. I did not consider `ao` a dependency as it is not passed down.

I will add the startTime back.

Reply:
Yeah, I agree that currently it's not being passed as a dependency downstream but we are looking at refactoring this code and when we do (it might take a while to be honest), this might change and even in general defining the start time should be independent of the analytics object.